internals/secrethub/mkdir.go

```diff
@@ -33,12 +33,14 @@ func NewMkDirCommand(io ui.IO, newClient newClientFunc) *MkDirCommand {
 }
 
 // Register registers the command, arguments and flags on the provided Registerer.
-func (cmd *MkDirCommand) Register(r command.Registerer) {
+func (cmd *MkDirCommand) Register(r cli.Registerer) {
 	clause := r.Command("mkdir", "Create a new directory.")
-	clause.Arg("dir-paths", "The paths to the directories").Required().PlaceHolder(dirPathsPlaceHolder).SetValue(&cmd.paths)
```

Comment:
It's not instantly obvious from this code, but this accepts multiple arguments.

You can do `secrethub mkdir path/a path/b`.

The new code does not yet support this.

Reply:
I was not aware of that. Thanks for telling me.

Reply:
Fixed.